src/core/worker.js

```diff
@@ -197,6 +197,115 @@ IPDFStreamRangeReader.prototype = {
 }
 
 /** @implements {IPDFStream} */
+class PDFNetworkStream {
+  constructor(data, msgHandler) {
+    this._data = data;
```

Comment:
Nit: Underscore is usually used to denote a property, or method, that should be considered "private". However, in both `PDFNetworkStreamReader` and `PDFNetworkStreamRangeReader` below, you're *directly* accessing `_data` (and* `_msgHandler`). To me, this feels like abusing notation, and it'd be better it you change the properties to `this.data = data` (and `this.msgHandler`) here instead. (Another option would be to keep them "private" here, and add getters.)